src/Rimu.jl

```diff
@@ -3,7 +3,7 @@ module Rimu
 using Arrow: Arrow
 using DataFrames: DataFrames, DataFrame, metadata
 using DataStructures: DataStructures
-using LinearAlgebra: LinearAlgebra, dot, isdiag
+using LinearAlgebra: LinearAlgebra, dot, isdiag, eigen, ishermitian
```

Comment:
Are `eigen` and `ishermitian` needed in the toplevel module?

In general, I'm not sure how I feel about explicitly using each function.